launcher/src/main/java/org/apache/spark/launcher/SparkLauncher.java

```diff
@@ -70,6 +70,8 @@
   public static final String EXECUTOR_EXTRA_JAVA_OPTIONS = "spark.executor.extraJavaOptions";
   /** Configuration key for the executor native library path. */
   public static final String EXECUTOR_EXTRA_LIBRARY_PATH = "spark.executor.extraLibraryPath";
+  /** Configuration key for the number of executor instances. */
+  public static final String EXECUTOR_INSTANCES = "spark.executor.instances";
```

Comment:
Can you use this constant in other places? For example, `core/src/main/scala/org/apache/spark/internal/config/package.scala`

Reply:
Actually in my scenario 
```suggestion
                sparkLauncher.setConf("spark.executor.instances", executors);
		sparkLauncher.setConf(SparkLauncher.EXECUTOR_MEMORY, executorMemory);
		sparkLauncher.setConf(SparkLauncher.DRIVER_MEMORY, driverMemory );
		sparkLauncher.setConf(SparkLauncher.EXECUTOR_CORES, executorCores);
```
There is no option in SparkLauncher for executors.